amplify-material-ui/src/auth/index.tsx

```diff
@@ -1,4 +1,5 @@
-export { default as Authenticator } from './authenticator';
-export { default as SignIn } from './sign-in';
+export * from './authenticator';
+export * from './sign-in';
 export { default as Loading } from './loading';
```

Comment:
Still default in export of "Loading"-Component

maybe:
export * from './loading';

Reply:
You're right. I will fix it